starter-repos/app-template/skaffold-ci.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: skaffold/v2beta7
+kind: Config
+metadata:
+  name: app-template
+build:
+  artifacts:
+  # Make sure the skaffold "default-repo" is set.
+  # Follow the doc for instructions https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/master/docs/index.md#testing-the-change-locally-with-skaffold 
+  - image: app
```

Comment:
Do you know when and where the `image` field will be updated? (`app` seems just to be a placeholder)

Reply:
It's updated by `skaffold` both locally (dev) and in the CI pipelines.